docs/user_guide/models/methylvi.md

```diff
@@ -0,0 +1,161 @@
+# MethylVI
+
+**methylVI** [^ref1] (Python class {class}`~scvi.model.MethylVI`) is a generative model of scBS-seq data that can subsequently
+be used for many common downstream tasks.
+
+The advantages of methylVI are:
+
+-   Comprehensive in capabilities.
+-   Scalable to very large datasets (>1 million cells).
+
+The limitations of methylVI include:
+
+-   Effectively requires a GPU for fast inference.
+-   Latent space is not interpretable, unlike that of a linear method.
+
+```{topic} Tutorials:
+
+-   Work in progress.
```

Comment:
As soon as the model is merged into scvi-tools I can add tutorial notebooks from [our original repository](https://github.com/suinleelab/methylVI) to the scvi-tools repo.

Reply:
Missed this one. In that case we can merge as soon as categorical_covariates are added to the codebase. Looks like all else is fixed.